modules/react-native-wallet/src/GooglePushProvisioning.ts

```diff
@@ -0,0 +1,82 @@
+import {NativeModules} from 'react-native';
+import type {NativeModule} from 'react-native';
+
+type Tsp = 'VISA' | 'MASTERCARD';
+
+type UserAddress = {
+    name: string;
+    addressOne: string;
+    addressTwo: string;
+    locality: string;
+    administrativeArea: string;
+    countryCode: string;
+    postalCode: string;
+    phoneNumber: string;
+};
+
+type PushTokenizeRequest = {
+    opc: string;
+    tsp: Tsp;
+    clientName: string;
+    lastDigits: string;
+    address: UserAddress;
+};
+
+type GooglePushProvisioningType = NativeModule & {
+    getTokenStatus: (tsp: Tsp, tokenReferenceId: string) => Promise<number>;
+    getActiveWalletID: () => Promise<string>;
+    getStableHardwareId: () => Promise<string>;
+    getEnvironment: () => Promise<string>;
+    pushProvision: (opc: string, tsp: Tsp, clientName: string, lastDigits: string, addressJson: string) => Promise<string>;
+};
+
+const GooglePushProvisioning = (NativeModules.GooglePushProvisioning as GooglePushProvisioningType) ?? null;
```

Comment:
I think there is a huge problem with native module linking. The `GooglePushProvisioning` module is always null when I try to run your code. How about throwing an error in this situation?

```suggestion
const GooglePushProvisioning = NativeModules.GooglePushProvisioning
  ? NativeModules.GooglePushProvisioning
  : new Proxy(
      {},
      {
        get() {
          throw new Error(LINKING_ERROR);
        },
      }
    );
```

Reply:
@brunovjk now the Apple error is thrown on Android devices 😅